main.go

```diff
@@ -146,3 +155,42 @@ func initLog() {
 	}
 	logrus.SetFormatter(formatter)
 }
+
+// define lxcfs processe.
+func setLxcfsProcess(processes exec.Processes) exec.Processes {
+	if !cfg.IsLxcfsEnabled {
+		return processes
+	}
+
+	lxcfsProcess := &exec.Process{
+		Path: cfg.LxcfsBinPath,
+		Args: []string{
+			cfg.LxcfsHome,
+		},
+	}
+	processes = append(processes, lxcfsProcess)
+
+	if cfg.LxcfsHome[len(cfg.LxcfsHome)-1:] == "/" {
+		cfg.LxcfsHome = cfg.LxcfsHome[0 : len(cfg.LxcfsHome)-2]
```

Comment:
What does the mean ???

Reply:
Remove the last char '/' to let `path.Dir` get the parent directory of lxcfs home

Reply:
You can use "Trim" .